src/org/traccar/Context.java

```diff
@@ -516,6 +516,10 @@ public static void init(IdentityManager testIdentityManager) {
         identityManager = testIdentityManager;
     }
 
+    public static void initMediaManager(MediaManager mediaManager) {
```

Comment:
This should be part of `init`.

Reply:
What do you mean on "part"? It is only used in the unittests. In init we have: mediaManager = new MediaManager(config.getString("media.path")); I could change it to initMediaManager(new MediaManager(config.getString("media.path"))); but I don't think that's what you meant.

Reply:
I want to have single `init` for unit tests.

Reply:
why? most tests don't need everything. isn't it a waste of time/resources?

Reply:
It's easier to maintain and creating a single empty object doesn't really affect performance or resource usage.